src/display/annotation_layer.js

```diff
@@ -448,7 +448,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
 
       let font = null;
       if (this.data.fontRefName) {
-        font = this.page.commonObjs.getData(this.data.fontRefName);
+        font = this.page.commonObjs.getSafely(this.data.fontRefName);
```

Comment:
While I totally agree that this code looks quite weird, I'd nonetheless prefer to not start making larger annotationLayer changes *here* since that feels like scope creep.

However, I'm not entirely happy about needing the `getSafely` method, so I'll change this to a combination of `PDFObjects.{has, get}` here instead to minimize the size of the patch.